README.md

```diff
@@ -55,7 +55,7 @@ Invoke-CommandAs -Session $PSSession -ScriptBlock { Get-Process }
 Invoke-CommandAs -Session $PSSession -ScriptBlock { Get-Process } -AsSystem -RunElevated
 
 # Execute Remotely on multiple Computers at the same time.
-Invoke-CommandAs -ComputerName 'VM01', 'VM02' -Credential $Credential -ScriptBlock { Get-Process }
+Invoke-CommandAs -ComputerName 'VM01', 'VM02' -AsUser $Credential -ScriptBlock { Get-Process }
 
```

Comment:
-Credential was the correct line here. Please revert. as this is the credentials to establish the initial psremote session